src/splat/segtypes/common/c.py

```diff
@@ -380,13 +380,16 @@ def get_c_line_include_macro(
             # IDO uses the asm processor to embeed assembly, and it doesn't require a special directive to include symbols
             asm_outpath = asm_out_dir / self.name / f"{sym.filename}.s"
             rel_asm_outpath = os.path.relpath(asm_outpath, options.opts.base_path)
-            return f'#pragma GLOBAL_ASM("{rel_asm_outpath}")'
+            final_path = Path(rel_asm_outpath).as_posix()
+            return f'#pragma GLOBAL_ASM("{final_path}")'
 
         if options.opts.use_legacy_include_asm:
             rel_asm_out_dir = asm_out_dir.relative_to(options.opts.nonmatchings_path)
-            return f'{macro_name}(const s32, "{rel_asm_out_dir / self.name}", {sym.filename});'
+            final_path = f"{(rel_asm_out_dir / self.name).as_posix()}"
```

Comment:
and this ^ (and the one on line 391 one) I dont think need to be f-strings?

Reply:
Learn something new every day 😊